docs/user-manuals/multi-hierarchy-elastic-quota-management.md

```diff
@@ -0,0 +1,519 @@
+# Multi-Hierarchy-ElasticQuota-Management
+
+Multi-Hierarchy-ElasticQuota-Management is an ability of koord-scheduler to manage different user's resource usage in a shared-cluster.
+
+## Introduction
+When several users or teams share a cluster, fairness of resource allocation is very important. This proposal provides
+multi-hierarchy elastic quota management mechanism for the scheduler. 
```

Comment:
This proposal provides
multi-hierarchy elastic quota management mechanism for the scheduler.
-->
Koordinator provides
multi-hierarchy elastic quota management mechanism in the scheduler.

Reply:
fixed